src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs

```diff
@@ -50,7 +51,12 @@ public RavenAwsGlacierClient(GlacierSettings glacierSettings, Progress progress
             else
                 credentials = new SessionAWSCredentials(glacierSettings.AwsAccessKey, glacierSettings.AwsSecretKey, glacierSettings.AwsSessionToken);
 
-            _client = new AmazonGlacierClient(credentials, region);
+            _client = new AmazonGlacierClient(credentials, new AmazonGlacierConfig
+            {
+                RegionEndpoint = region,
+                Timeout = RequestExecutor.GlobalHttpClientTimeout
```

Comment:
How is the user supposed to be able to configure that? Do we allow that?

Reply:
What if they need more? This is a `static readonly` value, and we might need a configurable option.

Reply:
We never allowed that, but I've added a configuration option now.